### PR TITLE
Type check decorators

### PR DESCRIPTION
**Pull request by tk0miya**
Note: This contains https://github.com/kaigionrails/conference-app/pull/97

### CODE REVIEW

File: sig/handwritten/app/decorators/profile_decorator.rbs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ProfileDecorator : Profile
```
Comment (tk0miya) [inline sig/handwritten/app/decorators/profile_decorator.rbs:1]: The component after `:` is called `module-self-types`. It describes what classes or modules will include this module. It tells Steep to this module can use the methods of "includer" 's.
https://github.com/ruby/rbs/blob/master/docs/syntax.md#declarations

In this case, decorator modules can access the methods of the decorated object.

**Comment by unasuke** [comment]
@tk0miya Would you rebase it from the main branch?

**Comment by tk0miya** [comment]
rebased.

**Review by unasuke — APPROVED** [review]
🙌

**Pull request merged**